gluten-core/pom.xml

```diff
@@ -221,7 +221,41 @@
         <version>${os-maven-plugin.version}</version>
       </extension>
     </extensions>
+    <resources>
+      <resource>
+        <filtering>true</filtering>
+        <!-- Include the properties file to provide the build information. -->
+        <directory>${project.build.directory}/extra-resources</directory>
+      </resource>
+    </resources>
     <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-antrun-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>build-info</id>
+            <goals>
+              <goal>run</goal>
+            </goals>
+            <phase>generate-resources</phase>
+            <configuration>
+              <target>
+                <exec executable="bash" osfamily="unix">
+                  <arg value="${project.basedir}/../dev/gluten-build-info.sh"/>
+                  <arg value="${project.basedir}/../ep/build-velox/build/velox_ep"/>
```

Comment:
Is it more appropriate to config BACKEND_TYPE and BACKEND_HOME in velox profile properties, and replace params with ${BACKEND_TYPE} and ${BACKEND_HOME}